android/app/build.gradle.kts

```diff
@@ -133,6 +133,10 @@ sentry {
   ignoredVariants.set(listOf("debug", "fast"))
 
   autoInstallation.sentryVersion.set(libs.versions.sentry.sdk.get())
+
+  sizeAnalysis {
+    enabled.set(true)
```

Comment:
explicitly set to `true` right now since the plugin doesn't have it set to true by default yet